ui/notification/system.go

```diff
@@ -0,0 +1,35 @@
+package notification
+
+import (
+	"path/filepath"
+
+	"github.com/gen2brain/beeep"
+)
+
+const (
+	icon  = "ui/assets/decredicons/qrcodeSymbol.png"
+	title = "Godcr"
+)
+
+type SystemNotification struct {
+	iconPath string
+	message  string
+}
+
+func NewSystemNotification() *SystemNotification {
+	absolutePath, err := getAbsolutePath()
+	if err != nil {
+		log.Error(err.Error())
+	}
+
+	return &SystemNotification{
+		iconPath: filepath.Join(absolutePath, icon),
+	}
+}
+
+func (s *SystemNotification) Notify(message string) {
+	err := beeep.Notify(title, message, s.iconPath)
+	if err != nil {
+		log.Info("could not initiate desktop notification, reason:", err.Error())
```

Comment:
This error should be returned not logged.

Reply:
The log avoids handling notification errors in the ui code. All that needs to be called is the Notify method, just like toast notifications.